Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2
# #175 Characters born year 20 or earlier displays age "???"

Resolves #175 


## Changelog:
- Characters born year 20 or earlier displays age "???"


## Developer changelog:
- Edited the vanilla age text to only display if birth year is greater than 20
- Added new text box to display when birth year is lesser or equal to 20
    - The text is "???"
    - Custom popup

# How to test
    - Make sure no ages over 562 are shown (in the 582 startdate). Check all UI that shows age.

# TODO
- ~~Char select screen~~
